static/app/components/charts/useChartZoom.tsx

```diff
@@ -78,51 +62,19 @@ interface Props {
  * the props that would be passed to the `BaseChart` as zoomRenderProps.
  */
 export function useChartZoom({
```

Comment:
this hook is only used in one place currently, took it as an opportunity to slim it down. The old ChartZoom did a lot to support onRestore which is not used anywhere